modules/hardware-qualcomm.nix

```diff
@@ -70,6 +75,12 @@ in
         quirks.fb-refresher.enable = true;
       };
     }
+    {
+      mobile = mkIf cfg.qualcomm-sdm845.enable {
+        system.system = "aarch64-linux";
+        quirks.fb-refresher.enable = true;
```

Comment:
TODO:

 - [ ] Verify if it changes anything with the current implementation

Though it doesn't cause any harm.